Sources/Services/ContainerAPIService/Containers/ContainersService.swift

```diff
@@ -457,15 +450,23 @@ public actor ContainersService {
 
         await self.exitMonitor.stopTracking(id: id)
 
-        // Try and shutdown the runtime helper.
-        do {
-            self.log.info("Shutting down sandbox service for \(id)")
+        // Shutdown and deregister the sandbox service
+        self.log.info("Shutting down sandbox service for \(id)")
 
-            let client = try state.getClient()
-            try await client.shutdown()
-        } catch {
-            self.log.error("failed to shutdown sandbox service for \(id): \(error)")
-        }
+        let path = self.containerRoot.appendingPathComponent(id)
+        let bundle = ContainerClient.Bundle(path: path)
+        let config = try bundle.configuration
+        let label = Self.fullLaunchdServiceLabel(
+            runtimeName: config.runtimeHandler,
+            instanceId: id
+        )
+
+        let client = try state.getClient()
+        try await client.shutdown()
```

Comment:
Sorry, I still think shutdown shouldn't be fatal and we can leave what we had as it's mostly best effort cleanup on the sb services end, I just meant to not ignore all of the errors like what you had in your original change.